docs/grammar.md

```diff
@@ -38,6 +39,8 @@ resourceDecl -> decorator* "resource" IDENTIFIER(name) interpString(type) "exist
 
 moduleDecl -> decorator* "module" IDENTIFIER(name) interpString(type) "=" (ifCondition | object | forExpression) NL
 
+testDecl -> "test" IDENTIFIER(name) interpString(type) "=" (object) NL
```

Comment:
You can define an object with specific properties (looks like this one may have `params` and/or `mocks`), but it's fine to leave as-is. I mostly asked because I wasn't sure what the object was for since I hadn't seen Anthony's gist yet.

Reply:
Sounds good